src/datagen/java/blusunrize/immersiveengineering/data/Recipes.java

```diff
@@ -1232,12 +1232,32 @@ private void mineralMixes(@Nonnull Consumer<FinishedRecipe> out)
 				.build(out, toRL("mineral/mephitic_quarzite"));
 		MineralMixBuilder.builder(nether)
 				.addNetherSpoils()
-				.addOre(Blocks.GRAVEL, .6f)
-				.addOre(Blocks.ANCIENT_DEBRIS, .4f)
+				.addOre(Blocks.POLISHED_BLACKSTONE_BRICKS, .4f)
+				.addOre(Blocks.POLISHED_BLACKSTONE, .3f)
+				.addOre(Blocks.ANCIENT_DEBRIS, .2f)
+				.addOre(Blocks.GILDED_BLACKSTONE, .1f)
```

Comment:
I don't love the reduction in Ancient Debris output here.
Is there a reason other than "making space for everything else"? I must admit, I have not *played* with it, so maybe it produces ridiculous amounts of Ancient Debris?

Reply:
Because, when I was playing around, the ancient debris haul was still _huge_ - that 0.5 instead of 0.7 on fail chance actually means it's closer to 0.7x instead of 0.5x the amount of debris. It was still a huge amount for me, especially compared to the rarity of debris in the nether.

The below image is two minutes (approximate) of runtime on an archaic digsite of 99.99% saturation
![image](https://user-images.githubusercontent.com/40431605/187919919-59f074b9-aac1-4b58-92ae-488f47412074.png)